src/main/java/creativei/service/impl/ReminderServiceImpl.java

```diff
@@ -1,7 +1,40 @@
 package creativei.service.impl;
 
+import creativei.dao.ReminderCustomDao;
+import creativei.dao.ReminderDao;
+import creativei.entity.Reminder;
+import creativei.exception.InvalidParamRequest;
+import creativei.service.ReminderService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import util.StringUtil;
 
 @Service("ReminderService")
-public class ReminderServiceImpl {
+public class ReminderServiceImpl implements ReminderService{
+    @Autowired
+    ReminderCustomDao reminderCustomDao;
+
+    Logger logger= LoggerFactory.getLogger(ReminderServiceImpl.class);
```

Comment:
Use logger for both info and error logs in the methods

Reply:
resolved in commit 18a0688